content/en/service_management/incident_management/response_team.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Incident Response Team
+further_reading:
+- link: "/service_management/incident_management/incident_settings/responder-types"
+  tag: "Documentation"
+  text: "Customize responder types in Incident Settings"
+---
+
+## Overview
+
+Form your response team by adding other users and assigning them responder types (responder roles) so they know what they should focus on during the incident response.
+
+### Adding responders
+
+A responder is any Datadog user who participates in the response process for a particular incident.
+
+When you add a responder to an incident...
+
+* ...the responder is notified about the incident via email.
+* ...if the incident is private, the responder can see the incident in Datadog.
+* ...if the incident has a Slack channel attached, the responders is automatically added to the Slack channel.
+
+Datadog also automatically adds users as responders when...
+
+* ...they perform any action updating the incident, including writing to the incident timeline.
+* ...they are notified about the incident via a notification rule or manual incident notification.
+
+The **Response Team** tab of the Incident Details page records the time an individual was added to the incident’s response team. It also records the time the responder last took an action affecting the incident in Datadog, such as updating its attributes or writing to its timeline.
```

Comment:
```suggestion
The **Response Team** tab of the Incident Details page records the time an individual is added to the incident’s response team. It also records the time the responder takes an action affecting the incident in Datadog, such as updating its attributes or writing to its timeline.
```

Reply:
@estherk15 Thanks for the feedback. I know we try to avoid "time words," but I think `last took` more precisely captures the product behavior here. The product doesn't the time of every action that you take, it only records the time of _latest_ action you took. So if I updated an incident three times, only the time for the last one is actually recorded and visible to the user in the product on this page. The time recorded for an update overwrites any previously recorded time. Let me know if that makes sense. (Maybe there's a better way to describe this though.)

Reply:
Thanks @cperryk, that makes sense, I assumed the former where it records every action.